zio-http/shared/src/main/scala/zio/http/Handler.scala

```diff
@@ -549,7 +547,7 @@ sealed trait Handler[-R, +Err, -In, +Out] { self =>
     self(in)
 
   final def sandbox(implicit trace: Trace): Handler[R, Response, In, Out] =
-    self.mapErrorCause(Response.fromCause(_))
+    self.mapErrorCauseZIO(c => ErrorResponseConfig.configRef.get.map(Response.fromCause(c, _)).flip)
```

Comment:
There's a fair amount of overhead here--of course, there was before, but now there is even more.

Could be useful to unsafely get the fiber ref value in as many places as you can "safely" bundle it up in lazy evaluation (like here).

Reply:
I tried to figure out, what you mean. But I don't get it.
Currently, this config is saved within a fiber ref. Therefore idk how to get it unsafe. Also idk if the call to sandbox needs to be that optimized? Don't we advertise this as a non-prod feature?
But what you try to say might be true for other places?

One other design option I thought of was, to use a ConcurrentHashMap with the route pattern as a key. That way we could avoid fiber refs and zio based access to our config storage.

Reply:
Ah right, FiberRef instead of Ref.

Reply:
Yes. That you can override the behavior per route/request.